.gitlab-ci.yml

```diff
@@ -123,7 +129,7 @@ test:behat:
   - mv features/*.png web/screenshots/
   - echo "- Screenshots are available here :" && cd web/screenshots && ls -h *.png | xargs -i echo "${CI_ENVIRONMENT_URL}/screenshots/{}"
   tags:
-    - review-apps-3
+    - review-apps-4
```

Comment:
Needs follow-up to make configurable

Reply:
Already on it, I made some tests with UI varaible, and also with file but didn't find a way yet to use current dotfile

Will continue on it soon

Reply:
Fixed with #137 137